content/300-guides/050-database/300-seed-database.mdx

```diff
@@ -161,7 +161,7 @@ To seed your database with TypeScript:
      "name": "my-project",
      "version": "1.0.0",
      "scripts": {
-       "ts-node": "ts-node --compiler-options '{\"module\":\"CommonJS\"}'"
+       "ts-node": "ts-node --compiler-options '{\\\"module\\\":\\\"commonjs\\\"}'"
```

Comment:
Interesting number of slashes 😄 
Good to know!

Reply:
Hey man, i dont make the rules 😁

Reply:
The capitalization of `CommonJS` does not matter here?